pkgs/applications/networking/cluster/fluxcd/default.nix

```diff
@@ -55,7 +55,10 @@ buildGoModule rec {
     '';
     homepage = "https://fluxcd.io";
     license = licenses.asl20;
-    maintainers = with maintainers; [ jlesquembre ];
+    maintainers = with maintainers; [
+      jlesquembre
+      superherointj
+    ];
     platforms = platforms.unix;
```

Comment:
```suggestion
```
buildGoModule defaults to a subset of unix.

Reply:
Done